backend/typescript/rest/givecloud.ts

```diff
@@ -1,5 +1,6 @@
 import { Router, Request, Response } from "express";
 import { body, validationResult } from "express-validator";
+import * as crypto from "crypto";
```

Comment:
Only import the createHmac function from crypto to reduce bundle size. Use `import {createHmac} from "crypto";`